.github/workflows/2-2-delete.yml

```diff
@@ -128,23 +138,16 @@ jobs:
         env:
           GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
-  finish_exercise:
-    name: Finish Exercise
-    needs: [find_exercise, post_review_content]
-    uses: skills/exercise-toolkit/.github/workflows/finish-exercise.yml@v0.4.0
-    with:
-      issue-url: ${{ needs.find_exercise.outputs.issue-url }}
-
-  disable_workflow:
-    name: Disable this workflow
-    needs: [find_exercise, post_review_content]
-    runs-on: ubuntu-latest
-
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v4
-      - name: Disable current workflow
-        run: gh workflow disable "${{github.workflow}}"
+      - name: Create comment - add step content
+        run: |
+          gh issue comment "$ISSUE_URL" \
+            --body-file "$NEXT_REVIEW_FILE"
         env:
           GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
+      - name: Disable current workflow and enable next one
+        run: |
+          gh workflow disable "Step 2-2"
+          gh workflow enable "Step 3-1"
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
@ilan-klinghofer just wanted to draw your attention to updates I made to this workflow to allow for hand off to lesson 3. Lmk if you have questions, otherwise I can merge this later today